resources/cdk/photo_asset_manager/SPECIFICATION.md

```diff
@@ -1,130 +1,128 @@
 # Photo Asset Management Tech Spec
 
-This is the Tech Spec for the Photo Asset Management cross services example. This doc details the specifics of the deliverable app.
+This is the tech spec for the Photo Asset Management cross-service example. This doc details the specifics of the deliverable app.
 
-## Reading this document
+## How to read this document
 
-This document is broken down into the following sections
+This contains the following sections
 
 * Introduction - A summary of app functionality.
-* Development / Deployment - Information on where to find CDK deployment and development instructions.
-* Front-end - Describes the resources used to create the front-end, and how it interacts with the back-end.
-* Back-end - Describes the resources used to create the back-end.
-    * Common - Explains the concept of common resources vs language specific and describes the common resources.
-    * Language specific - Explains the programming language specific resources.
+* Development / Deployment - Information on where to find AWS Cloud Development Kit (AWS CDK) deployment and development instructions.
+* Frontend - Describes the resources used to create the frontend, and how the frontend interacts with the backend.
+* Backend - Describes the resources used to create the backend.
+    * Common - Explains the concepts of common resources and language-specific resources. Also describes the common resources.
+    * Language-specific - Explains the programming language specific resources.
 
-* READMES - Defines the information needed for each language specific readme.
+* READMES - Defines the information needed for each language-specific readme.
 * Glossary - Lists common terms.
 * Appendices
     * Appendix A - Policy details
     * Appendix B - Lambda triggers
```

Comment:
This is a great opportunity to use in-page anchors. LMK if this would be a good addition and I can add it in the separate PR.